MinPlatformPkg/Test/Library/TestPointCheckLib/MmCheckSmrr.c

```diff
@@ -1,12 +1,13 @@
 /** @file
```

Comment:
Same request to remove formatting only changes.

Reply:
Fixed.